src/openforms/scss/pdfs/_submission-step-row.scss

```
@@ -47,6 +47,7 @@
   &__value {
     @include body();
     width: 60%;
+    word-break: break-word;
```

`break-word` is deprecated

```suggestion
    word-break: break-all;
```